### PR TITLE
feat(cfb): add seperation between archived and active forms

### PR DESCRIPTION
**Pull request by Yelinz**
image

Add separate list for archived forms and labels for the form state

### CODE REVIEW

**Comment by anehx** [comment]
Hm maybe tabs (or a button group filter) would be better? I mean what's the use case for wanting to see all forms at once?

**Comment by Yelinz** [comment]
> Hm maybe tabs (or a button group filter) would be better? I mean what's the use case for wanting to see all forms at once?

image

I dont see the use case aswell and it looks nicer now

**Comment by Yelinz** [comment]
image
This is how the filters look like

File: addon/components/cfb-form-list/item.js [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
-  layout,
+export default class componentsCfbFormItemListItem extends Component {
+  @service intl;
 
-  tagName: "li",
+  layout = layout;
 
-  classNames: ["uk-text-nowrap", "uk-flex", "uk-flex-middle"],
-});
+  tagName = "li";
```
Comment (anehx) [inline addon/components/cfb-form-list/item.js:10]: Use a glimmer component so this won't be necessary, this component is easy to migrate..

**Review by anehx — CHANGES_REQUESTED** [review]

File: addon/components/cfb-form-list/item.js (hunk 1 above)
Comment (Yelinz) [reply addon/components/cfb-form-list/item.js]: Changed

**Comment by czosel** [comment]
Tests are failing ...

**Review by anehx — APPROVED** [review]

**Pull request merged**